nupic/research/frameworks/vernon/run_experiment/trainables.py

```diff
@@ -424,3 +429,115 @@ def _save(self, checkpoint_dir):
 
     def _restore(self, checkpoint):
         pass
+
+
+def run(config):
```

Comment:
The run functions seem to be added make in. You should delete them and make sure they haven't been called anywhere.